### ISSUE TITLE
Resurrect separate sview build and packaging for slurm

### ISSUE THREAD

**Issue by koomie** [openhpc/ohpc#1289]

**Comment by koomie** [comment]
We had this previously at some point (#679) but didn't make it to current 2.x build.

**Comment by koomie** [comment]
Separate sview package will be available in 2.1 release:

```# zypper info slurm-sview-ohpc
Loading repository data...
Reading installed packages...


Information for package slurm-sview-ohpc:
-----------------------------------------
Repository     : OpenHPC-2 - Updates
Name           : slurm-sview-ohpc
Version        : 20.11.3-13.1.ohpc.2.1
Arch           : x86_64
Vendor         : obs://private/OpenHPC:2.1:Factory
Installed Size : 2.4 MiB
Installed      : No
Status         : not installed
Source package : slurm-ohpc-20.11.3-13.1.ohpc.2.1.src
Summary        : Graphical user interface to view and modify Slurm state
Description    : 
    This package provides sview, which can be used to view Slurm configuration, job,
    step, node and partitions state information. Authorized users can also modify
    select information.
```

**Comment by koomie** [comment]
New check added to test suite passing successfully in CI.
